packages/yaml/src/index.ts

```diff
@@ -0,0 +1,143 @@
+import { type Service } from '@volar/language-service';
+import { type TextDocument } from 'vscode-languageserver-textdocument';
+import { type LanguageSettings, type LanguageService } from 'yaml-language-server';
+import { getLanguageService } from 'yaml-language-server/lib/umd/languageservice/yamlLanguageService.js';
+
+export interface Provide {
+	'yaml/languageService': () => LanguageService;
+}
+
+function isYaml(document: TextDocument): boolean {
+	return document.languageId === 'yaml';
+}
+
+function noop(): undefined { }
+
+/**
+ * Create a Volar language service for YAML documents.
+ */
+export function createYamlService(
```

Comment:
This is different from the other services by the way. Now this can be used as such from ESM:

```js
import { createYamlService } from 'volar-service-yaml'

createYamlService()
```

as opposed to:

```js
import createTypeScriptService from 'volar-service-typescript'

createTypeScriptService.default()
```

This relates o #46

Reply:
I'm +1 on this and would love to see this change in other services

Reply:
I agree, but it's inconsistent with other services. Let's unify changes for all services in a separate commit/PR.